scripts/emulator_install_apk.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -e
+
+source ./scripts/emulator_read_target.sh
+
+echo -e "\nInstalling APKs..."
+i=0
+step_complete=false
+until [[ $step_complete == true ]]; do
+    if adb -s emulator-5554 install -r -t -d --abi "${abi}" "android/app/build/outputs/apk/debug/app-${abi}-debug.apk" |& tee "${output_dir}/emulator_install_apk.txt"; then
```

Comment:
path could be wrong if script is not run from root directory. solve with `git rev-parse --show-toplevel` or error if not in root